js/apps/account-ui/src/oid4vci/Oid4Vci.tsx

```diff
@@ -0,0 +1,204 @@
+import {
+    Select,
+    SelectList,
+    SelectOption,
+    PageSectionVariants,
+    PageSection,
+    ActionList,
+    ActionListItem,
+    List,
+    ListItem,
+    MenuToggleElement,
+    MenuToggle
+} from '@patternfly/react-core';
+import { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useAlerts, useEnvironment } from "@keycloak/keycloak-ui-shared";
+import { usePromise } from "../utils/usePromise";
+import { Page } from "../components/page/Page";
+import { parseResponse } from "../api/parse-response";
+import { token } from '../api/request';
+
+type CredentialsIssuer = {
+  credential_issuer: string;
+  credential_endpoint: string;
+  authorization_servers: string[]; 
+  credential_configurations_supported: Map<string, SupportedCredentialConfiguration>
+}
+
+type SupportedCredentialConfiguration = {
+  id: string,
+  format: string,
+  scope: string
+}
+
+type VCState = {
+  dropdownItems: string[],
+  selectOptions: Map<string, SupportedCredentialConfiguration>,
```

Comment:
Same here, this should be a `Record`.
```suggestion
  selectOptions: Record<string, SupportedCredentialConfiguration>,
```